src/purging_rmw.c

```diff
@@ -246,16 +246,30 @@ get_then_time(struct dirent *entry, const char *entry_path)
  */
 int
 purge (const st_waste * waste_curr)
+{
+  return empty_option_purge(waste_curr, 0);
```

Comment:
A wrapper function isn't really necessary or desirable in this case. See my comments [on this post](https://github.com/theimpossibleastronaut/rmw/issues/229#issuecomment-506867694) about the use of the 'extern' keyword. using "empty_trash" instead of "want_empty" for the var name is fine though.

Then you can just use "empty trash" at the top of purge()